docs/react/toggling-devices.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 4
+---
+
+# Toggling devices
+
+Fishjam Cloud provides functionality to control media device streams more dynamically. This includes both soft control, such as muting, and hard control, like physically turning the device on or off.
+
```

Comment:
What do you think about adding a sentence or two that describes why we should consider using one method over another?

Something like this: 

Turning the device off releases resources, which saves battery and CPU power, and turns off the LED if there is one. However, turning it back on is slower, so other participants need to wait longer to receive the first data. You would likely use this method with a camera, as it’s usually acceptable to wait longer to see someone’s face; we typically don’t toggle our cameras as often during meetings. A disabled camera indicator reassures the user that the camera is truly turned off.

On the other hand, muting and unmuting the device is much faster, but a muted device still consumes resources. However, we don’t waste time starting the device when unmuting. This method is particularly useful for microphones, as it is common practice to mute and unmute multiple times during a meeting. Unmuting needs to be quick to capture the first word of a sentence.

Reply:
d